packages/api/tests/endpoints/test-granules.js

```diff
@@ -503,6 +540,58 @@ test.serial('GET returns the expected existing granule', async (t) => {
   t.deepEqual(response.body, expectedGranule);
 });
 
+test.serial('GET returns the expected existing granule if a collectionId is provided', async (t) => {
```

Comment:
We need a test re: the failure message/case branch logic that was added.  Also: see comment about possibly seperating out the method.

Reply:
I _think_ we have the error cases covered? I have...

- Test for the collectionId in the wrong format: https://github.com/nasa/cumulus/pull/2978/files/1fdde4b88ed0b14711daeee7e16f57c810f31d92#diff-cb59b4f181e7f47dd6a5f459c15ec0e0d45fff7fa4c83f378424e55256fd61a7R571

- Test for the collection not found: https://github.com/nasa/cumulus/pull/2978/files/1fdde4b88ed0b14711daeee7e16f57c810f31d92#diff-cb59b4f181e7f47dd6a5f459c15ec0e0d45fff7fa4c83f378424e55256fd61a7R583
-  Test for the granule not found: https://github.com/nasa/cumulus/pull/2978/files/1fdde4b88ed0b14711daeee7e16f57c810f31d92#diff-cb59b4f181e7f47dd6a5f459c15ec0e0d45fff7fa4c83f378424e55256fd61a7R595

Is there something you saw missing?

Reply:
Agree re ^, resolving.